frontend/pages/ForgotPasswordPage/ForgotPasswordPage.tsx

```diff
@@ -59,13 +59,13 @@ const ForgotPasswordPage = () => {
               target="_blank"
               rel="noopener noreferrer"
             >
-              Password reset FAQ
+              Password reset FAQ&nbsp;
+              <img
+                className="icon-external"
+                src={ExternalLinkIcon}
+                alt="Open external link"
+              />
```

Comment:
Instead of using `&nbsp;` to create the space, please create a global `icon-external` class that has `left-padding` or `left-margin` applied. That way, the space can be created once instead of every time it's implemented.

Reply:
The first solution uses &nbsp;

The second solution uses adding span to the last word of every anchor link

I tried using margin-right: -15px, but that only messes up the following text after the anchor link.

Reply:
I pulled it down and it looks like even with &nbsp; it's still wrapping the image. Are you seeing something different? 

![image](https://user-images.githubusercontent.com/2495927/187547879-793efaec-a630-4776-ad4d-6e880a3f5098.png)